decidim-core/app/cells/decidim/card_g/show.erb

```diff
@@ -0,0 +1,18 @@
+<%= link_to resource_path, class: classes[:default], id: resource_id do %>
+  <% if has_image? %>
+    <div class="<%= classes[:img] %>">
+      <%= image_tag resource_image_path, alt: title %>
+    </div>
+  <% end %>
+
+  <div class="<%= classes[:text] %>">
+    <%= content_tag title_tag, title, class: title_class %>
+    <%= description if has_description? %>
+
+    <% if metadata_cell.present? && !metadata_cell.blank? %>
```

Comment:
This condition is redundant?

Reply:
I've renamed the `has_description?` method with `show_description?`. Even if the resource has a description, it only has to be shown under the conditions contained in that method (highlighted)

Reply:
There is no point in using ```metadata_cell.present? && !metadata_cell.blank?``` 
Either use ```if metadata_cell.present? ... ```, either use ``` unless metadata_cell.blank?``` 

https://guides.rubyonrails.org/active_support_core_extensions.html#blank-questionmark-and-present-questionmark

Reply:
I can't remember why I put that condition, but I totally agree that it doesn't make sense, sorry.